### PR TITLE
feat(web-analytics): Add a modal, to allow users to see a larger version of a query

### PR DESCRIPTION
**Pull request by robbie-c**
## Problem
The queries have a limited number of rows, limited space, etc, so add the ability to make them bigger (in a modal).

## Changes

https://github.com/PostHog/posthog/assets/2056078/502d583f-66f1-497b-923d-fddb19b98108

## How did you test this code?

All manually

### CODE REVIEW

File: frontend/src/scenes/web-analytics/WebPropertyFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { PropertyFilters } from 'lib/components/PropertyFilters/PropertyFilters'
+import { isEventPropertyOrPersonPropertyFilter } from 'lib/components/PropertyFilters/utils'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+
+import { WebAnalyticsPropertyFilters } from '~/queries/schema'
+
+export const WebPropertyFilters = ({
```
Comment (robbie-c) [inline frontend/src/scenes/web-analytics/WebPropertyFilters.tsx:7]: this is just moved, not changed

File: frontend/src/scenes/web-analytics/WebAnalyticsTile.tsx [hunk 2]
```diff
@@ -379,3 +379,22 @@ const getBreakdownValue = (record: unknown, breakdownBy: WebStatsBreakdown): str
     }
     return breakdownValue
 }
+
+export const WebQuery = ({
```
Comment (robbie-c) [inline frontend/src/scenes/web-analytics/WebAnalyticsTile.tsx:383]: also moved

File: frontend/src/queries/nodes/DataTable/queryFeatures.ts [hunk 3]
```diff
@@ -59,5 +60,9 @@ export function getQueryFeatures(query: Node): Set<QueryFeature> {
         features.add(QueryFeature.resultIsArrayOfArrays)
     }
 
+    if (isWebStatsTableQuery(query)) {
+        features.add(QueryFeature.hideFooterIfCantLoadMore)
+    }
+
```
Comment (mariusandra) [inline frontend/src/queries/nodes/DataTable/queryFeatures.ts:66]: This seems like a very very very specific feature 🤔

**Review by mariusandra — COMMENTED** [review]
I wonder why you didn't just make the button a link to `urls.insightNew(..., query)` that just opens in a new window? It's equally full screen _and_ allows you to dive even deeper into the data if you want to.

File: frontend/src/queries/nodes/DataTable/queryFeatures.ts (hunk 3 above)
Comment (robbie-c) [reply frontend/src/queries/nodes/DataTable/queryFeatures.ts:66]: > This seems like a very very very specific feature 🤔

untitled(1)

It's to hide this^ which is imo quite confusing atm. I think there's been a few situations like, where I want to make slightly different UI decisions in web analytics than were made in insights, but don't want to do a full rewrite. E.g. adding VizSpecificOptions here https://github.com/PostHog/posthog/blob/f0c4908aba87f1713d305ed50f4d5165f4784837/frontend/src/queries/schema.ts#L467

Comment (Twixes) [reply frontend/src/queries/nodes/DataTable/queryFeatures.ts:66]: We should definitely change the word "events" to "entries" in that message. Feel free to do here.

Comment (robbie-c) [reply frontend/src/queries/nodes/DataTable/queryFeatures.ts:66]: > I wonder why you didn't just make the button a link to urls.insightNew(..., query) that just opens in a new window? It's equally full screen and allows you to dive even deeper into the data if you want to.

There's a few kinks to iron out before WebStatsTableQuery nodes can be opened as a new insight. I'm closing this PR in favour of https://github.com/PostHog/posthog/pull/20148

**Comment by robbie-c** [comment]
Closing in favour of https://github.com/PostHog/posthog/pull/20148

**Pull request closed without merging**